docs/documentation/full-text/sorting.mdx

```diff
@@ -123,7 +123,107 @@ LIMIT 5;
+
+When using `ORDER BY` with multiple sort fields, ParadeDB can now partially push down the sorting operation. In this case, only the first column is pushed down to the BM25 index, and PostgreSQL handles the additional columns using sort operations.
+
+For example, in the following query with multiple sort fields, sorting by `sale_date` is pushed down to the BM25 index, while sorting by `amount` is handled by PostgreSQL:
+
+```sql
+SELECT description, sale_date, amount, paradedb.score(id)
+FROM sales
+WHERE description @@@ 'laptop'
+ORDER BY sale_date, amount
+LIMIT 10;
+```
+
+You can verify if partial ORDER BY pushdown occurred by running `EXPLAIN` on the query. The query plan will show a `Custom Scan` with our ParadeDB scan provider, followed by an appropriate sort operation based on your PostgreSQL version:
+
+- In PostgreSQL 16+: Often uses an `Incremental Sort` node which can take advantage of the already-sorted first column
+- In older PostgreSQL: Uses a regular `Sort` node, but still benefits from our optimized ordering
+
+<Accordion title="Example Query Plan">
+```csv
+                                                                              QUERY PLAN
+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------
+ Limit  (cost=0.19..0.34 rows=10 width=44) (actual time=0.708..0.709 rows=0 loops=1)
+   ->  Incremental Sort  (cost=0.19..0.34 rows=10 width=44) (actual time=0.707..0.708 rows=0 loops=1)
+         Sort Key: sale_date, amount
+         Presorted Key: sale_date
+         Full-sort Groups: 1  Sort Method: quicksort  Average Memory: 25kB  Peak Memory: 25kB
+         ->  Custom Scan (ParadeDB Scan) on sales  (cost=0.00..0.03 rows=10 width=44) (actual time=0.702..0.703 rows=0 loops=1)
+               Table: sales
+               Index: sales_index
+               Segment Count: 1
+               Heap Fetches: 0
+               Exec Method: NormalScanExecState
+               Scores: true
+               Tantivy Query: {"with_index":{"oid":631060,"query":{"parse_with_field":{"field":"description","query_string":"keyboard","lenient":null,"conjunction_mode":null}}}}
+ Planning Time: 1.343 ms
+ Execution Time: 1.167 ms
+(15 rows)
+```
+</Accordion>
+
+This feature significantly improves performance when sorting by multiple columns, as the index is used for the first level of sorting, requiring PostgreSQL to perform less work to produce the final ordered results.
+
+<Note>
+Limitations for partial `ORDER BY` pushdown:
+
+1. Only the first sort field is pushed down to the BM25 index.
+2. The first sort field must be indexed as a fast field.
+3. A `LIMIT` clause is still required.
+</Note>
+
+## Sorting With Score Retrieval
+
+ParadeDB now supports optimized sorting even when retrieving scores. This enables efficient sorting while still benefiting from score retrieval, which is a common use case in search applications.
```

Comment:
"now" would be good in a blog post, rather than the docs.
```suggestion
ParadeDB supports optimized sorting even when retrieving scores. This enables efficient sorting while still benefiting from score retrieval, which is a common use case in search applications.
```

Reply:
You're right about removing "now" - it's better for documentation to be timeless. Fixed.